### ISSUE TITLE
Add collectibles from Island Expeditions

### ISSUE THREAD

**Issue by SacredDuckwhale** [SacredDuckwhale/Rarity#49]
Roadmap:
- [x] Research: What is the drop rate? Does it change with difficulty? Higher if you win?
- [x] Research: Do pets drop from ALL scenarios? Or are they based on the expedition type (as most pets seem to refer to specific factions or races that can be found there)?	
- [x] Feature: Add Island Expedition tracking (scenario completion or "rewards screen"?)
- [x] Task: Add toys to DB
- [x] Task: Add pets to DB
- [x] Testing: Verify scenario completion tracking (or rewards screen if that is easier)
- [x] Testing: Verify attempts being added for all relevant items

**Comment by SacredDuckwhale** [comment]
As for the drop rate: [Thread on Reddit](https://www.reddit.com/r/woweconomy/comments/99f7ys/island_expedition_pets/e4pjwgx/?context=3)

> I have 60+ victories on normal (two 3-man teams with one character playing on both teams) as all five of my characters have the ten win achievement. My main has won 20+ playing on both teams.
For those 60+ normal victories, I received two pets and 4 or 5 rep items. That is < 10 special rewards for 60ish wins. The satchels drop consistently and now that I have the order hall dubloon upgrade, I am getting 10+ from the bigger satchel.
I now have 12 Heroic victories and I have seen one rep item drop. Nothing else. Not much data but I have the feeling that the mounts could be a nightmare. At least the pets are tradeable.
tldr; 60+ normal victories --> less that 10 special rewards (4 or 5 rep tokens and 2 pets) 12 heroic victories --> 1 special reward (rep token)

A 1-3% drop rate may be a first starting point. Unfortunately, it is unclear whether the difficulty affects drop chances, the type of island has any effect, or even if there is some sort of loot lockout in place...

**Comment by SacredDuckwhale** [comment]
Added in 1cce6ccfc4628c2bb6b63d345a841e68dfa15d16
